docs/integrator/macros.md

```diff
@@ -41,8 +41,12 @@ Replaces macros in a given string with their corresponding values.
 | {player.widthInt}        | The current player width as int       |
 | {player.heightInt}       | The current player height as int      |
 | {player.duration}        | The duration of current video *       |
-| {player.durationInt}     | The duration as int *                  |
+| {player.durationInt}     | The duration as int *                 |
```

Comment:
what is the purpose of the '*' at the end of some of the macro descriptions? is that just for padding?

Reply:
it's referencing a note / qualification on line 65:
> \* Returns 0 if video is not loaded. Be careful timing your ad request with this macro.

Reply:
Thank you 😁